.github/workflows/build.yml

```
@@ -47,123 +85,42 @@ jobs:
-          ARCH=amd64
-          BRANCH=${{ env.deps_version_branch }}
+    - name: Build and publish image
+      env:
+        REPO: docker.io/longhornio
+        TAG: ${{ needs.build_info.outputs.image_tag }}-amd64
+        TARGET_PLATFORMS: linux/amd64
+      run: make workflow-image-build-push
 
   build-push-arm64-images:
     name: Build and push ARM64 images
     runs-on: longhorn-infra-oracle-arm64-runners
```

actionlint (1.7.4)

label "longhorn-infra-oracle-arm64-runners" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-22.04", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "macos-12-xl", "macos-12-xlarge", "macos-12-large", "macos-12", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)